bosh-setup/azuredeploy.json

```diff
@@ -113,10 +113,10 @@
 
     "baseUriAzureCloud": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/bosh-setup/",
```

Comment:
I understand I can give the default value to the parameter. But from our point of view, we don't want to expose the `baseUriAzureCloud` as a parameter to users because most end-users don't need to change the url.
If this best practice is not a **MUST**, could you please merge the PR at this point? And we can investigate whether we should expose it or not. If users ask for it, we can expose it in future.